cms/djangoapps/contentstore/views/item.py

```diff
@@ -552,7 +552,10 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
         category = dest_usage_key.block_type
 
         # Update the display name to indicate this is a duplicate (unless display name provided).
-        duplicate_metadata = own_metadata(source_item)
+        duplicate_metadata = {}
```

Comment:
It wasn't an issue before because most XBlocks have simple types for their settings. LibraryContent blocks use a unique `LibraryList` value type which contains `LibraryVersionReference` objects which contain `LibraryLocator` and `ObjectId` instances.

Three representations are:
* JSON String: `'[["library-v1:ProblemX+PR0B", "5436ffec56c02c13806a4c1b"], ...]'` (`unicode` type)
* Intermediate: `[["library-v1:ProblemX+PR0B", "5436ffec56c02c13806a4c1b"], ...]`(`list` type)
* Native: `[(LibraryLocator(org="ProblemX", library="PR0B"), ObjectId("5436ffec56c02c13806a4c1b")), ...]` (`LibraryList` type)

Unfortunately it's easy to mix and match the intermediate and the native types for most XBlocks - but for this one in particular it breaks things. Part of the issue is the API design of `from_json` and `to_json` - if they always took and returned *strings*, there would be no confusion, but since they take and return intermediate python types which resemble native types 99% of the time, we get this situation. I guess that the intermediate types should never really be used outside of the XBlock internal/runtime/fieldstore context but it appears that they are.